source/docs/README.md

```diff
@@ -1,6 +1,6 @@
 # DevOps Step-by-step guides
-|  [**Create an Azure Subscription free account**](https://github.com/MohamedRadwan-DevOps/devops-step-by-step/blob/main/source/docs/create-an-azure-subscription-free-account.md) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| ![enter image description here](https://raw.githubusercontent.com/MohamedRadwan-DevOps/devops-step-by-step/main/source/docs/images/icons/azure-50.png) |
+|  [**Create an Azure Subscription free account**](https://github.com/MohamedRadwan-DevOps/devops-step-by-step/blob/main/source/docs/source/create-an-azure-subscription-free-account.md) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| ![enter image description here](https://raw.githubusercontent.com/MohamedRadwan-DevOps/devops-step-by-step/main/source/docs/images/icons/azure-50.png) |
```

Comment:
Please change the absolute path for all links to the relative path.

For example instead of using this:

https://github.com/MohamedRadwan-DevOps/devops-step-by-step/blob/main/source/docs/source/create-an-azure-subscription-free-account.md

Use this one:
./source/create-an-azure-subscription-free-account.md

Reply:
@AhalimZaki thanks for your contribution @RehabAbotalep thanks for your excellent notice and comment! 👌